Detectors/MUON/MCH/GlobalMapping/include/MCHGlobalMapping/ChannelCode.h

```diff
@@ -46,6 +47,7 @@ class ChannelCode
 {
  public:
   ChannelCode() = default;
+  ChannelCode(const ChannelCode& cc) : mValue{cc.value()} {}
```

Comment:
is it needed to explicitly declare a copy constructor? If so, not explicitly declaring also an assignment operator can issue a compilation warning (-Wdeprecated-copy-with-user-provided-copy)

Reply:
removed